src/lib/parsers/SettingResolver.js

```diff
@@ -14,9 +14,10 @@ class SettingResolver extends Resolver {
 	 * @param {string} name The name of the key being resolved
 	 * @returns {string}
 	 */
-	async user(data, guild, name) {
+	async user(data, guild, name, { resolve, def }, silent) {
```

Comment:
Missing docs for resolve, def, and silent. Also, I assume this breaks the "all optional parameters go in an options object as the last parameter" consistency rule.

Same goes for the many instances of the same pattern below.

Reply:
This will probably be change because I don't like this format at all.